src/lib/realmdb/RealmDB.js

```diff
@@ -402,6 +407,27 @@ class RealmDB implements DB, ProfileDB {
   async deleteProfile(): Promise<boolean> {
     return this.profiles.deleteOne({ user_id: this.user.id })
   }
+
+  /**
+   *
+   * @param {TransactionData} data
+   * @returns {Promise<any>}
+   */
+  // eslint-disable-next-line require-await
+  async addToOutbox(data: TransactionData): Promise<any> {
+    return this.inboxes.insertOne({ user_id: this.user.id, ...data })
+  }
+
+  /**
+   *
+   * @param {string} txHash
+   * @returns {Promise<any>}
+   */
+  async getFromOutbox(txHash: string): Promise<any> {
```

Comment:
should return just the TransactionData + change return type

Reply:
Changed in f3caaed8bdbc62f3ba1a4c852c87f10d2168c0df